### ISSUE TITLE
Newtonsoft.Json.dll incompatible with 2019.2.3f1

### ISSUE THREAD

**Issue by sam598** [KhronosGroup/UnityGLTF#487]
I receive 26 errors when opening the latest project with Unity 2019.2.3f1 on OSX. The issue seems to be with Newtonsoft.json.

`Assets/UnityGLTF/Scripts/Tests/Editor/GLTFAssetGeneratorTests.cs(3,7): error CS0246: The type or namespace name 'Newtonsoft' could not be found (are you missing a using directive or an assembly reference?)`

I've tried building with 4.5 as a target, swapping out for the latest version of Newtonsoft.Json.dll, and https://github.com/jilleJr/Newtonsoft.Json-for-Unity same errors every time.

2019.1.2f1 works fine right out of the box.

**Comment by maxdepondt** [comment]
Same here. 
I also tried with 2019.2.1
Same problem

**Comment by jaykumarthaker** [comment]
Download [latest version](https://github.com/JamesNK/Newtonsoft.Json/releases) (or the .net version matches to your requirements)  and replace Newtonsoft.Json.dll

**Comment by cwmorse** [comment]
I also had 26 errors opening the latest project with 2019.2.8f1, so I'm assuming it's the same problem. Downloading the latest version of newtonsoft per above suggestion did not seem to solve the issue.

For what it's worth, I noticed that all the errors were in scripts in the Assets/UnityGLTF/Scripts/Tests/Editor folder. I don't know how 'Editor' folders work, but a dummy script referencing newtonsoft outside of this folder did not have a problem, only scripts inside the folder were unable to find it.

Since it was called 'Tests', I guessed it wasn't a required folder and just deleted it. I'm still able to import .glb files in the editor and still able to load .gltf files at runtime through the GLTFComponent. Haven't tried building the project yet, but it looks to all be working in editor.
